### ISSUE TITLE
Performance of closedSet

### ISSUE THREAD

**Issue by rumusan** [dimitymiller/cac-openset#4]
Thank you for the work! 

I found the performace of "closedSet" is much better than the "softmax" in paper. These two have any difference?

**Comment by dimitymiller** [comment]
The "softmax" result in the paper is taken from the established benchmark results published in earlier papers in the area.

In actuality, I also noticed that our closed-set softmax results could achieve higher than had been reported in these earlier papers, especially on TinyImageNet. I believe this is because previous papers may not have trained their closed-set classifiers to convergence.

**Comment by dimitymiller** [comment]
I'm going to close this issue, but if you want to discuss in more detail email me at dimity.miller@hdr.qut.edu.au
